src/main/java/net/minecraftforge/server/command/CommandDimensions.java

```diff
@@ -46,7 +46,7 @@
             .executes(ctx -> {
                 ctx.getSource().sendFeedback(new TranslationTextComponent("commands.forge.dimensions.list"), true);
                 final Registry<DimensionType> reg = ctx.getSource().func_241861_q().func_243612_b(Registry.field_239698_ad_);
-
+                
```

Comment:
you added a tab here, remove it to clean the diff

Reply:
argh, i've changed something here to test the dimension registry but reverted it later, apparently eclipse replaced the spaces with tabs, i'm sorry, i already changed settings to only use spaces in future on that project.